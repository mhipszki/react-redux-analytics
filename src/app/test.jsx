import React from 'react';
import { shallow } from 'enzyme';

import { App } from './';

test('renders prev / next buttons and a page', () => {
  const component = shallow(<App />);
  expect(component).toMatchSnapshot();
});
