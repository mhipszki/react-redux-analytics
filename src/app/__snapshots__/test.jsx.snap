// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders prev / next buttons and a page 1`] = `
<row__Row>
  <column__Column>
    <nav__Nav
      disabled={false}
    >
      Prev
    </nav__Nav>
  </column__Column>
  <column__Column
    style={
      Object {
        "alignSelf": "flex-start",
      }
    }
  >
    <Slide />
  </column__Column>
  <column__Column>
    <nav__Nav
      disabled={false}
    >
      Next
    </nav__Nav>
  </column__Column>
</row__Row>
`;
